pkg/common/configs/configvalidator_test.go

```diff
@@ -1910,3 +1910,80 @@ func TestCheckLimitsStructure(t *testing.T) {
 	assert.Equal(t, len(partitionConfig.Queues[0].Limits), 0)
 	assert.Equal(t, len(partitionConfig.Limits), 0)
 }
+
+
+func TestCheckQueuesStructure(t *testing.T) {
```

Comment:
We can also check the content of the error message. And if possible, it would be great to verify whether rootQueue has been correctly added to the PartitionConfig as well.

Reply:
I update the code, check the content of the error message, and check the rootQueue with assert.